### ISSUE TITLE
Request for Blender Simulation Scripts/Parameters

### ISSUE THREAD

**Issue by dcolli23** [columbia-ai-robotics/garmentnets#5]
# Description

Hello! My name is Dylan Colli and I'm looking into utilizing GarmentNets to proof-of-concept a deformable object tracker. Part of that would require doing my own blender simulations of the grasping in Blender. 

I see in the paper that the simulation was done with a randomly chosen grasp point 21 times for each garment instance, so I'm assuming this was done via Blender's scripting interface. Is this assumption correct? If so, is it possible to release the scripts and parameters/blend files used to generate the Blender cloth simulations? I'm not too well-versed in Blender simulation and it would be a huge help to have these files.

If Blender's scripting API wasn't used, would it be possible to still release the .blend files used for these simulations? I'm less concerned about obtaining the actual 3D model (as I know they're available as part of the CLOTH3D dataset) and I'm more interested in the parameters used to make the cloth simulation look more physically plausible.

# Requests

- Is it possible to release the .blend files used for simulation?
    - If not, can the cloth simulation parameters be released?
- Can the scripts used to generate Blender cloth simulations be added to this repository?

Thanks a ton and congrats on the super interesting paper!

# Side Note

P.s. if Cheng is the one who is reading this, I wanted to say thanks for the work on CDCPD2. I'm currently in Dmitry's lab working on extending that to multi-template tracking. I thought you would want to know that your contribution to CDCPD is still doing great work and it's still being used.

**Comment by cheng-chi** [comment]
Hi Dylan,
   Indeed, the GarmetNets data was simulated and rendered in blender using its python API. Since all objects are generated on-the-fly with python script, we don't have blend files avaliable. I have shared the repo that I used to do reserach with blender with your GitHub account. Unfortunately it wasn't maintained for years and therefore might not work with new version of blender and other python packages, but I hope it is helpful. The relevant entrypoint is `scripts/smpl_cloth_grip/smpl_cloth_dataset_generation_v2.py`.
